src/neuroconv/datainterfaces/ecephys/spikeglx/spikeglxdatainterface.py

```diff
@@ -146,7 +148,9 @@ def get_device_metadata(self) -> dict:
         return device
 
 
+# include for backwards compatibility
 class SpikeGLXLFPInterface(SpikeGLXRecordingInterface):
+    pass
```

Comment:
Did you want to eventually deprecate it altogether? If so just need a warning here